ts-obs/cmd/timescaledbGetPassword.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+    "errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// timescaledbGetPasswordCmd represents the timescaledb get-password command
+var timescaledbGetPasswordCmd = &cobra.Command{
+	Use:   "get-password",
+	Short: "Gets the TimescaleDB/PostgreSQL password for a specific user",
+	RunE:  timescaledbGetPassword,
+}
+
+func init() {
+	timescaledbCmd.AddCommand(timescaledbGetPasswordCmd)
+    timescaledbGetPasswordCmd.Flags().StringP("user", "u", "postgres", "user whose password to get")
```

Comment:
do we have more than 1 user in the secret now? Should we? @blagojts maybe you can weigh in?

Reply:
We currently have a postgres user and an admin user in the secret.